### PR TITLE
chore: Included githubactions in the dependabot config

### PR DESCRIPTION
**Pull request by naveensrinivasan**
This should help with keeping the GitHub actions updated on new releases. This will also help with keeping it secure.

Dependabot helps in keeping the supply chain secure https://docs.github.com/en/code-security/dependabot

GitHub actions up to date https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot

https://github.com/ossf/scorecard/blob/main/docs/checks.md#dependency-update-tool
Signed-off-by: naveensrinivasan <172697+naveensrinivasan@users.noreply.github.com>

### CODE REVIEW

**Comment by naveensrinivasan** [comment]
@MounirDhahri 👀

**Review by MounirDhahri — APPROVED** [review]
LGTM 
Thanks again @naveensrinivasan for your contribution 🙏. I left one small comment and we can merge this

File: .github/dependabot.yml [hunk 1]
```diff
@@ -33,3 +33,13 @@ updates:
...
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule: 
+      interval: daily
+    open-pull-requests-limit: 0
+    assignees: 
+     - brainbicycle
+     - pvinis
+    reviewers: 
+     - artsy/collector-experience
```
Comment (MounirDhahri) [inline .github/dependabot.yml:45]:
```suggestion
     - artsy/mobile-platform
```

Comment (MounirDhahri) [reply .github/dependabot.yml:45]: Our collector experience team no longer owns the infra updates, our newly created mobile platform team takes care of it now

**Comment by naveensrinivasan** [comment]
> LGTM
> Thanks again @naveensrinivasan for your contribution 🙏. I left one small comment and we can merge this

PTAL. Thanks

**Review by MounirDhahri — APPROVED** [review]
Nice, thanks 🛳🛳🛳

**Pull request merged**